### PR TITLE
Test against different Symfony versions

### PR DESCRIPTION
**Pull request by WouterJ**
This also tests the claim to support Symfony 3

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -25,10 +31,12 @@ env:
 
 before_install:
   - composer self-update
+  - if [ "$SYMFONY_VERSION" != "" ]; then composer require --no-update symfony/symfony:$(SYMFONY_VERSION); fi;
 
 install:
   - if [ "$deps" = "no" ]; then composer install; fi;
   - if [ "$deps" = "low" ]; then composer --prefer-lowest --prefer-stable update; fi;
+  - if [ "$deps" = "update" ]; then composer --prefer-dist update; fi;
```
Comment (stof) [inline .travis.yml]: no need for ``deps=update``: we don't have a lock file in the repo, so install and update are doing the same already

**Comment by WouterJ** [comment]
On hold until https://github.com/doctrine/doctrine2/pull/1543 is merged and a new version is released

**Comment by weaverryan** [comment]
@WouterJ doctrine/doctrine2#1543 was merged and is now part of 2.5.2. So, this can be pushed forward again, right?

**Comment by WouterJ** [comment]
Currently waiting for Security ACL 3.0.0-BETA1 to be released (seems to be forgotten).

**Comment by weaverryan** [comment]
@WouterJ I sent a pull request to make symfony/security-acl an optional dependency to the cache bundle: https://github.com/doctrine/DoctrineCacheBundle/issues/76

**Comment by WouterJ** [comment]
Tests are green now, ready to be merged.

**Comment by weaverryan** [comment]
:+1:

**Comment by xabbuh** [comment]
What about removing the build for PHP 5.6 without any special Symfony version? This would save one job per build.

File: .travis.yml [hunk 2]
```diff
@@ -18,16 +18,24 @@ matrix:
...
+      env: SYMFONY_VERSION="2.3.*"
+    - php: 5.6
+      env: SYMFONY_VERSION="2.8.*@dev" SYMFONY_DEPRECATIONS_HELPER=strict
+    - php: 5.6
+      env: SYMFONY_VERSION="3.0.*@dev" SYMFONY_DEPRECATIONS_HELPER=strict
 
 env:
   global:
     - deps=no
+    - SYMFONY_DEPRECATIONS_HELPER=weak
```
Comment (stof) [inline .travis.yml]: strict should be the default, and weak should be used for deps=low only

Comment (WouterJ) [inline .travis.yml]: fixed.

**Comment by stof** [comment]
Thank you @WouterJ.

**Pull request closed without merging**